documentation/source/_static/css/custom.css

```diff
@@ -2,9 +2,12 @@
 .wy-side-nav-search>div.version{color:#000000;}
 .wy-menu-vertical a:visited{color:#d9d9d9;}
 .wy-menu-vertical p.caption{color:#3185bc;}
-.wy-nav-content {max-width: none;}
+/* .wy-nav-content {max-width: none;} */
```

Comment:
I wanted to point out this change I made (thinner content section), because it wasn't requested in the TODO list.

This is an aesthetic change (personal preference) so I'm happy to put it back. But, I wanted to get feedback on this change. I felt it puts us more in line with other Python packages on RTD, but perhaps there was a good reason for having a wider content section.

Reply:
yes, i am the one who introduced the no-width limit. My argument is that nowadays people have big screens and I am much more comfortable reading this:

![image](https://user-images.githubusercontent.com/2482071/115771119-8dc90b00-a37b-11eb-8d2d-bbb2836dde9f.png)

compared to that:
![image](https://user-images.githubusercontent.com/2482071/115771161-9883a000-a37b-11eb-9a73-5d50f19607d7.png)

This a similar debate about the 80col limitation for coding, and i am usually the one pushing for having more than 80col 😉 

I agree that most RTD pages use a max-width, but i'm pretty sure the main reason being that it is set by default and people did not bother (or knew) to remove that limitation.

Reply:
Ah, that's a very good point. I think the only way it would be feasible is if all of the left/right aligned graphics are changed, so because of that, I'll undo the change.